src/main/java/guru/qa/allure/notifications/clients/rocket/RocketClient.java

```diff
@@ -0,0 +1,49 @@
+package guru.qa.allure.notifications.clients.rocket;
+
+import guru.qa.allure.notifications.clients.Notifier;
+import guru.qa.allure.notifications.config.rocket.Rocket;
+import guru.qa.allure.notifications.exceptions.MessagingException;
+import guru.qa.allure.notifications.json.JSON;
+import guru.qa.allure.notifications.template.RocketTemplate;
+import guru.qa.allure.notifications.template.data.MessageData;
+import java.io.ByteArrayInputStream;
+import kong.unirest.ContentType;
+import kong.unirest.Unirest;
+
+public class RocketClient implements Notifier {
+
+    private final JSON json = new JSON();
+    private final Rocket rocket;
+    private final RocketTemplate template;
+
+    public RocketClient(MessageData messageData, Rocket rocket) {
+        this.rocket = rocket;
+        this.template = new RocketTemplate(messageData);
+    }
+
+    @Override
+    public void sendText() throws MessagingException {
+        String body = String.format("{\"channel\": \"%s\", \"text\": \"%s\" }",
+            rocket.getChannel(), template.create().replace("\r\n", "\\\n"));
+        String url = String.format("%s/api/v1/chat.postMessage", rocket.getUrl());
```

Comment:
what if `rocket.getUrl()` ends with `/`?

Reply:
it works fine
<img width="1021" alt="image" src="https://github.com/qa-guru/allure-notifications/assets/36734883/f94abb1f-cb73-4759-8482-4ff7dc13468d">